R/FilterState.R

```diff
@@ -412,6 +432,8 @@ FilterState <- R6::R6Class( # nolint
     extract_type = logical(0),
     x_reactive = NULL, # reactive containing the filtered variable, used for updating counts and histograms
     filtered_na_count = NULL, # reactive containing the count of NA in the filtered dataset
+    disabled = FALSE, # is this filter disabled?
```

Comment:
default should be `NULL` as we use reactiveVal here
```suggestion
    disabled = NULL, # reactiveVal returning logical
```